src/test/java/org/folio/dew/BulkEditTest.java

```diff
@@ -250,6 +242,7 @@ void shouldUpdateProgressUponItemIdentifiersJob() throws Exception {
   @ParameterizedTest
   @EnumSource(value = IdentifierType.class, names = {"USER_NAME", "EXTERNAL_SYSTEM_ID", "INSTANCE_HRID", "ITEM_BARCODE", "ISSN", "ISBN"}, mode = EnumSource.Mode.EXCLUDE)
   @DisplayName("Run bulk-edit (item identifiers) successfully")
+  @Disabled
```

Comment:
@AntonAntonich  please remove @Disabled annotation. The same for other tests

Reply:
done